cldf/StructureDataset-metadata.json

```diff
@@ -9,25 +9,38 @@
     "dc:conformsTo": "http://cldf.clld.org/v1.0/terms.rdf#StructureDataset",
     "dc:identifier": "https://github.com/segbo-db/segbo",
     "dc:title": "SegBo",
+    "dcat:accessURL": "https://github.com/cldf-datasets/segbo",
     "prov:wasDerivedFrom": [
         {
-            "rdf:about": "git@github.com:glottolog/glottolog",
+            "rdf:about": "https://github.com/cldf-datasets/segbo",
             "rdf:type": "prov:Entity",
-            "dc:created": "v4.4-20-gf6531fcec3",
+            "dc:created": "4ba76fa",
+            "dc:title": "Repository"
+        },
+        {
+            "rdf:about": "https://github.com/glottolog/glottolog",
+            "rdf:type": "prov:Entity",
+            "dc:created": "v4.4-8-g43833512ad",
             "dc:title": "Glottolog"
+        },
+        {
+            "rdf:about": "https://github.com/segbo-db/segbo",
+            "rdf:type": "prov:Entity",
+            "dc:created": "v1.0.0-8-geb7c042",
+            "dc:title": "Repository"
```

Comment:
If `cldfbench` detects a git repository in `raw`, it automatically adds corresponding metadata to the CLDF metadata.